src/hooks/routing/useRouterTrade.ts

```diff
@@ -69,7 +70,7 @@ export function useRouterTrade<TTradeType extends TradeType>(
     provider: library as JsonRpcProvider,
   })
 
-  const { isFetching, isError, data, currentData } = useGetQuoteQuery(queryArgs ?? skipToken, {
+  const { isLoading, isError, data } = useGetQuoteQuery(queryArgs ?? skipToken, {
```

Comment:
hm so one small thing, data v. currentdata

>data - The latest returned result regardless of hook arg, if present.
currentData - The latest returned result for the current hook arg, if present.

one issue you might encounter is if queryargs change, data will not be invalidated, but currentdata will